app/controllers/course/controller.rb

```diff
@@ -59,4 +59,12 @@ def current_component_host
     @current_component_host ||= Course::ComponentHost.new(current_tenant.settings(:components),
                                                           current_course.settings(:components))
   end
+
+  before_action :add_course_breadcrumb, only: [:show, :index]
+
+  def add_course_breadcrumb
+    if current_course.present?
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.